<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1483</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="TitleFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Sans&quot;;
font-size: 28px;
border-radius: 5px;
font-weight:bold;
color: rgb(47, 172, 81);
</string>
     </property>
     <property name="text">
      <string>  Facial Expression Recognition</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="MenuBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="webcam">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;


</string>
       </property>
       <property name="text">
        <string>WebCam</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="video_btn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;


</string>
       </property>
       <property name="text">
        <string>Upload video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="image_button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;



</string>
       </property>
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;


</string>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="webc_frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>1191</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="img_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>581</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Sans&quot;;
font-size: 28px;
color:rgb(47, 172, 81);
border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Expression: </string>
     </property>
    </widget>
    <widget class="QLabel" name="expr_lbl">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="graph_close">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>390</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="MplWidget" name="MplWidget" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>551</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="webc_start">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>420</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>420</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="video_frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>1191</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="save_btn_v">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>420</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upload_btn_v">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="graph_close_v">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>390</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="expr_lbl_v">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="img_lbl_v">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>581</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Sans&quot;;
font-size: 28px;
color:rgb(47, 172, 81);
border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Expression: </string>
     </property>
    </widget>
    <widget class="MplWidget" name="MplWidget_v" native="true">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>551</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn_v">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>420</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="image_frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>1191</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="close_btn_i">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>420</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="expr_lbl_i">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="img_lbl_i">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>461</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Sans&quot;;
font-size: 28px;
color:rgb(47, 172, 81);
border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Expression: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_btn_i">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>420</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upload_btn_i">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 172, 81);
border-radius: 5px;
font-weight:bold;
color: rgb(255,255,255);
height:25px;
 border: none;
display: block;




</string>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <zorder>close_btn_i</zorder>
    <zorder>expr_lbl_i</zorder>
    <zorder>label_4</zorder>
    <zorder>save_btn_i</zorder>
    <zorder>img_lbl_i</zorder>
    <zorder>upload_btn_i</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="test1.qrc"/>
 </resources>
 <connections/>
</ui>
